shenyuClientRust_versioned_docs/version-1.0.0/http.md

```diff
@@ -0,0 +1,158 @@
+---
```

Comment:
Change this file name to "version-0.2.0", in crate, `1.0.0` is a stable version that can only be determined after long-term testing and verification of various functions. You can check the `mio` or `opendal` in https://crates.io